content/documentation.rst

```diff
@@ -0,0 +1,370 @@
+the page appears in the index documentation tree.
+
+
+Docstrings
+==========
+
+* Well documented code is extremely important.
+  Take time to describe components, how they work, their limitations, and the
+  way they are constructed.
+  Don't leave others in the team guessing what is the purpose of uncommon or
+  non-obvious code.
+
+* Document code as part of docstrings and not as comments.
+
+.. sourcecode:: python
+
+    def iamanExample(doc):
+        """
+        A simple docstring is placed here.
+        """
+          config = self.createSomethingHere('')
+
+There are two types of docstrings for production and test code.
+
+
+Test code docstrings
+--------------------
+
+Test code docstrings can contain information during the review process of new
+tests that can be written.
+
+.. sourcecode:: python
+
+    class TestHelpers(IAmATestCase):
+        """
+        The docstring here may add tests for helpers for a certain module
+        """
+        def test_of_a_module_1(self):
+         """
+         What is expected to happen in the first module of this test case
+         """
+        def test_of_a_module_2(self):
+         """
+         What is expected to happen in the second module of this first case
+         """
+
+
+Production code docstrings
+--------------------------
+
+Docstring are added in the production code to provide further information for
+readers and reviewers.  
+
+For example:
+
+.. sourcecode:: python
+
+    def getSomethingNewHere(self):
+
+Whereas a docstring should be added to add further information:
+
+.. sourcecode:: python
+
+    def getSomethingNewHere(self):
+        """
+        A docstring describing what SomethingNewHere is about
+        """
+
+
+Marketing / Promotional Materials
+=================================
+
+Promotions and marketing materials are located in the main website.
+
+It should be as generic and non-technical as possible with links to the
+Documentation for more in-depth / technical information.
+
+Please go to the internal wiki under General > Marketing for internal marketing
+details and links to image files.
+
+
+Announcing a new release to the email list
+------------------------------------------
+
+After the website is updated and News item published, we send a newsletter:
+
+1. Go to Campaigns in Mailchimp.
+
+2. Select 'Replicate' besides 'NEW: SFTPPlus Release Announcement'.
+   If it is a security bugfix, use the Security Advisories email list.
+
+3. Select the News Announcements email list.
+
+4. Update the email subject and email with the News text used to announce the
+   new release. You can use the text in the News article for the email.
+
+5. Select Send. Before sending the final email, preview first by going
+   to 'Preview and Send' on the top menu and select 'Send a test email'.
+   
+
+Known Issues
+============
+
+This section lists known issues for the current release of SFTPPlus along with
```

Comment:
This introductory sentence makes no sense.

I was expecting a sentence descriging why we need to know issues, what problem it tries to solve and how we solve it